control/control.yaml

```diff
@@ -107,3 +109,23 @@ components:
           type: string
           enum: [start, stop]
 
+    Control.Bgpv4ControlState:
+      description: >-
+        Control BGP Protocol state to handle special case.
+        Because generic protocol control handle within FlowTransmitState
+      type: object
+      required: [state]
+      properties:
+        names:
+          description: >-
+            The names of BGP object to control.
+          type: array
+          items:
+            type: string
+            x-constraint:
+            - '/components/schemas/Device.Bgpv4/properties/name'
+        state:
+          description: >-
+            The BGP Protocol state.
+          type: string
+          enum: [start, stop, link_flap]
```

Comment:
is link_flap the right term here for a BGP session?

Reply:
I have removed link_flap  and added withdraw_routes and advertise_routes which cover Sonic use case.